### PR TITLE
feat: remove some field validations for specific external course marketing types

### PR DESCRIPTION
**Pull request by AliAdnanSohail**
[Remove/Override Certificate data validation for EE Sprints and Stacks](https://2u-internal.atlassian.net/browse/PROD-3259)

### CODE REVIEW

File: course_discovery/apps/course_metadata/data_loaders/csv_loader.py [hunk 1]
```diff
@@ -278,6 +278,13 @@ def validate_course_data(self, course_type, data):
             required_fields.extend(
                 settings.CSV_LOADER_TYPE_SOURCE_REQUIRED_FIELDS[course_type.slug][self.product_source.slug]
             )
+            # Remove some fields for specific external course marketing type
+            external_course_marketing_type = data.get('external_course_marketing_type', '')
+            if external_course_marketing_type in [
+                ExternalCourseMarketingType.Sprint, ExternalCourseMarketingType.CourseStack
+            ]:
+                required_fields.remove('certificate_header')
+                required_fields.remove('certificate_text')
```
Comment (DawoudSheraz) [inline course_discovery/apps/course_metadata/data_loaders/csv_loader.py]: What about facts?

Comment (AliAdnanSohail) [reply course_discovery/apps/course_metadata/data_loaders/csv_loader.py]: As per requirements discussed on slack, we will just not get certificate info for these courses

**Review by DawoudSheraz — APPROVED** [review]

**Review by Ali-D-Akbar — APPROVED** [review]

File: course_discovery/apps/course_metadata/data_loaders/csv_loader.py (hunk 1 above)
Comment (DawoudSheraz) [reply course_discovery/apps/course_metadata/data_loaders/csv_loader.py]: Lets verify the facts usage internally.

Comment (AliAdnanSohail) [reply course_discovery/apps/course_metadata/data_loaders/csv_loader.py]: @DawoudSheraz I didn't get your point? What we need to verify?

Comment (DawoudSheraz) [reply course_discovery/apps/course_metadata/data_loaders/csv_loader.py]: That `facts` wont be there for mentioned marketing types.

**Pull request merged**